components/net_traits/lib.rs

```diff
@@ -91,6 +155,8 @@ pub struct LoadData {
     /// The policy and referring URL for the originator of this request
     pub referrer_policy: Option<ReferrerPolicy>,
     pub referrer_url: Option<Url>,
+    #[ignore_heap_size_of = "Defined in hyper"]
```

Comment:
Hyper has `HeapSizeOf` support, and that `RequestSource` enum isn't defined in Hyper.